pwndbg/commands/heap.py

```diff
@@ -371,3 +371,89 @@ def find_fake_fast(addr, size):
 
             if main_heap.fastbin_index(value) == fastbin:
                 malloc_chunk(start+offset-pwndbg.arch.ptrsize,fake=True)
+
+
+@pwndbg.commands.ParsedCommand
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWhenHeapIsInitialized
```

Comment:
Could you change it to use `ArgparsedCommand` instead? For the `addr` argument we can use https://github.com/pwndbg/pwndbg/blob/dev/pwndbg/commands/__init__.py#L259-L272